### DISCUSSION TITLE
disable csfr on .env or configuration file instead of every form?

### DISCUSSION THREAD

**Discussion by wnunezc** [symfony/symfony#44739] [Q&A] [answered]
hello, guys, i am afected by [44541](https://github.com/symfony/symfony/issues/44541) 
in all my forms i get this:

> The CSRF token is invalid. Please try to resubmit the form.

i seek that i need put this configuration in all form becouse i dont whant to downgrade to 5.4.0:

```php
$resolver->setDefaults([
            'data_class'      => Task::class,
            // enable/disable CSRF protection for this form
            'csrf_protection' => true,
            // the name of the hidden HTML field that stores the token
            'csrf_field_name' => '_token',
            // an arbitrary string used to generate the value of the token
            // using a different string for each form improves its security
            'csrf_token_id'   => 'task_item',
        ]);
```

this is tedious; Is there any way to disable CFSR tokens at a general symfony level, some unknown directive in for `.env` ???

I am in the development stage of a personal project that currently does not interest me to have active CFSR; Once the 5.4.2 version is ready, I will update and activate the tokens again ...

**Comment by lugosium** [comment, marked as answer]
You seem to be looking for https://symfony.com/doc/current/reference/configuration/framework.html#enabled

**Reply by wnunezc** [reply]
yes but it is missing a example on documentation ...
